database/models.py

```diff
@@ -40,6 +40,18 @@ class HarvestSource(db.Model):
         db.String(36), db.ForeignKey("organization.id"), nullable=False
     )
     frequency = db.Column(db.String, nullable=False)
+    frequency = db.Column(
+        Enum(
+            "manual",
+            "daily",
+            "weekly",
+            "biweekly",
+            "monthly",
+            name="frequency",
+        ),
+        nullable=False,
+        index=True,
+    )
```

Comment:
Aren't these incompatible? Does the second overwrite the first?

Reply:
ooh good catch! that should be a replace.